src/services/users/types.ts

```diff
@@ -348,6 +348,13 @@ export interface UserRoles {
   roles: ObjectId[];
 }
 
+export interface EmailTemplates {
```

Comment:
All instances of EmailTemplates could be changed to EmailTemplateConfigurations if we wanted to be verbose.

Reply:
Fine for me like this 👍